MaxText/configs/v4/22b.sh

```diff
@@ -20,7 +20,6 @@ echo "Running 22b.sh"
 # OUTPUT_PATH (Required, unless base_output_directory is already set in base.yml)
 # DATASET_PATH (Required, unless dataset_path is already set in base.yml)
 # RUN_NAME (Required, unless run_name is already set in base.yml or running with XPK/GKE)
-# PLATFORM (Optional, can be "gke" or "gce", default is "gce")
```

Comment:
Could you also remove the `export PLATFORM="gce"` in line 34?

Reply:
Done